### PR TITLE
Pull request for the Using Bootstrap project

### PR DESCRIPTION
**Pull request by euqueme**
to whom it concerns: 
I have a warning(not error) on line #815 because the Magazine is a section but its also just an image with no text.

Thank you in advance for your kind review 💃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1937 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <!-- Required meta tags -->
```
Comment (whiz25) [inline index.html]: remove comment

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1937 @@
...
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <!-- Bootstrap CSS -->
```
Comment (whiz25) [inline index.html]: this too

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Amazing job :+1:  @euqueme 
Your project looks awesome and am impressed with the effort you have put into it.
This project meets most of the requirements but we need to fix a few minor things before we can merge. Check my suggestions below
- [x] your navbar needs a fix below ```320px```. compare it the original site.
- [x] given that this is a clone, please add a ```hamburger menu``` for screens below ```768px```
- [x] remove all comments from your code.
- [x] clone the original site as close as possible.

Please send for another review when you done.

**Comment by euqueme** [comment]
to whom it concerns:
I didn't made the fix at 320px on the navbar originally because Bootstrap doesn't have a breakpoint at mentioned screen size so to accomplish this I had to use a media query.

Thank you in advance for your kind review 💃

**Review by whiz25 — APPROVED** [review]
## Awesome job :clap: @euqueme 
You have implemented all the suggested changes and therefore it's a :+1: from me.
Use ```Martin Afani``` when filling the form. Good luck in your up coming projects.

**Pull request merged**